corehq/tests/test_classcleanup.py

```diff
@@ -0,0 +1,142 @@
+from functools import wraps
+from unittest import TestCase
+
+from nose.plugins import PluginTester
+from testil import eq
```

Comment:
what is gained by using `eq` here vs relying on `TestCase`'s assertions? I remember you mentioned boilerplate, but everything is already part of a `TestCase`, and each test has `self` passed in. Is this just a preference for writing `eq(` rather than `self.assertEquals(`?

Reply:
I copied the pattern from [another plugin test](https://github.com/dimagi/commcare-hq/blob/f8e57b130b00e1d47bfdda5843fb3ce4a16021ba/corehq/tests/test_nose_timing.py). It's not a strong reason, `self.assertEquals()` would have worked too.